paddle/fluid/operators/controlflow/while_op.cc

```diff
@@ -22,6 +22,12 @@
 #ifdef PADDLE_WITH_MKLDNN
 #include "paddle/fluid/platform/mkldnn_helper.h"
 #endif
+
+PADDLE_DEFINE_EXPORTED_bool(
+    cache_inference_while_scope,
+    false,
+    "Cache the scope of while on inference to avoid create scope repeatedly.");
```

Comment:
Cache the scope of the while op to avoid repeated creation of the scope for each iteration and improve inference performance